framework/src/mesh/MooseMesh.C

```diff
@@ -136,11 +136,9 @@ MooseMesh::validParams()
                              "In particular you must supply this for GMSH meshes. "
                              "Note: This is completely ignored for ExodusII meshes!");
 
-  MooseEnum partitioning("default=-3 metis=-2 parmetis=-1 linear=0 centroid hilbert_sfc morton_sfc",
-                         "default");
   params.addParam<MooseEnum>(
       "partitioner",
-      partitioning,
+      partitioning(),
```

Comment:
This isn't necessary and I think makes reading the validParams method less clear, see my comment below in PatchSide...

Reply:
To make this clear, you suggest
```
params.addParam<MooseEnum>("partitioner", MooseMesh::partitioning(), "...");
params.set<MooseEnum>("partitioner") += "grid";
```

Is that correct?

Reply:
You don't need to add it, it is already there from the base class call

Reply:
No it's not? `MeshGenerator != MooseMesh`

Reply:
Ah wait, I see why you did this. I was thinking that PatchSidesetGenerator inherited from this class, but it does not.

Reply:
Alright - is this acceptable, then?

Reply:
yes